client/components/gsuite/gsuite-price/style.scss

```diff
@@ -1,4 +1,37 @@
-.gsuite-price__price-per-user {
+.gsuite-price__monthly-price {
+	font-size: 18px;
+	font-weight: 500;
+
+	strong {
+		font-size: 24px;
+		font-weight: normal;
+	}
+}
+
+.gsuite-price__annual-price {
+	color: var( --color-text-subtle );
+	display: inline-block;
+	font-size: 12px;
```

Comment:
We should match the size to the discounted price. It should be `14px` if we use the [badge component](https://wpcalypso.wordpress.com/devdocs/design/badge) for the discount.